scheduler/src/cook/mesos/util.clj

```diff
@@ -376,6 +376,23 @@
                  name-filter-fn (filter #(name-filter-fn (:job/name %))))
         (take limit)))))
 
+(defn committed?
```

Comment:
We already have [`uncommitted?`](https://github.com/twosigma/Cook/blob/5bb3c2cab43b1d63c8635cf28429173e06890cd8/scheduler/src/cook/mesos/util.clj#L384); is there an opportunity for code re-use between them?